src/wallet.test.ts

```diff
@@ -432,6 +432,50 @@ describe('wallet', () => {
       const promise = pify(engine.handle).call(engine, payload);
       await expect(promise).rejects.toThrow('Invalid input.');
     });
+
+    it('should not throw if verifyingContract is undefined', async () => {
+      const { engine } = createTestSetup();
+      const getAccounts = async () => testAddresses.slice();
+      const witnessedMsgParams: TypedMessageParams[] = [];
+      const processTypedMessageV3 = async (msgParams: TypedMessageParams) => {
+        witnessedMsgParams.push(msgParams);
```

Comment:
Nit: This variable seems to serve no purpose:
```suggestion
      const processTypedMessageV3 = async (msgParams: TypedMessageParams) => {
```